internal/debugger/tarball.go

```diff
@@ -66,8 +82,8 @@ func CreateTarball(ssm bool) {
 				fmt.Println("Debug logging enabled successfully.")
 			}
 
-			fmt.Println("\nWaiting 60 seconds to collect debug logs...")
-			showProgressBar(60)
+			fmt.Println("\nWaiting 90 seconds to collect debug logs...")
```

Comment:
I would pass the constant so that it will not be misleading if we update the constant
```suggestion
			fmt.Printf("\nWaiting %d seconds to collect debug logs...",logWaitTime)
```